tests/vinyldns_tests.bats

```diff
@@ -166,6 +159,21 @@ load test_helper
   [ "${output}" = "${fixture}" ]
 }
 
+@test "search-record-sets" {
```

Comment:
Do you think it's worth adding something like the following, so this is consistent with the style elsewhere?
```suggestion
@test "search-record-sets (when the search returns results)" {
```

Reply:
Added suggested context.